FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -619,4 +624,60 @@ - (FIRConfigUpdateListenerRegistration *)addOnConfigUpdateListener:
   return [self->_configRealtime addConfigUpdateListener:listener];
 }
 
+#pragma mark - Rollout
+
+- (void)addRemoteConfigInteropSubscriber:(id<FIRRolloutsStateSubscriber>)subscriber {
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:RolloutsStateDidChangeNotificationName
+                  object:self
+                   queue:nil
+              usingBlock:^(NSNotification *_Nonnull notification) {
+                FIRRolloutsState *rolloutsState =
+                    notification.userInfo[RolloutsStateDidChangeNotificationName];
+                [subscriber rolloutsStateDidChange:rolloutsState];
+              }];
+  // Pass active rollout metadata stored in persistence if exists while app launched.
+  NSArray<NSDictionary *> *activeRolloutMetadata = self->_configContent.activateRolloutMetadata;
+  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
I am not sure about the async/sync here.
When Crashlytics register with FirebaseSession, there is a sync call to send the first notification. Should we also follow this pattern?
https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseSessions/Sources/FirebaseSessions.swift#L253

Reply:
Updated to sync.